src/style.css

```diff
@@ -0,0 +1,157 @@
+* {
+  margin: 0;
+}
+
+/* .card__info-product {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--mainColor);
+  margin: 0 0 4px;
+} */
+
+/* p {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--mainColor);
+  margin: 0;
+} */
+
+:root {
+  --mainColor: #060b35;
+  --blueAccent: #00acdc;
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
Move this to `*` selector.

Reply:
All this code? it will add borders to all elements

Reply:
`box-sizing: border-box;` needed to display elements on the page correctly. It is good practice to add this property to all elements (using `*` selector)